### DISCUSSION TITLE
Copycon command

### DISCUSSION THREAD

**Discussion by jwillens** [community/community#23312] [New to GitHub]
<p>Excuse the newbie question, but going through the beginner courses here for Git on Windows, it says to use the “Copycon” command to create new files or repos. Yet, when I try this in any terminal setting, I get this:</p>
<p>copycon : The term ‘copycon’ is not recognized as the name of a cmdlet,<br>
function, script file, or operable program. Check the spelling of the name,<br>
or if a path was included, verify that the path is correct and try again.<br>
At line:1 char:1</p>
<ul>
<li>copycon test.txt</li>
<li>
<pre><code class="lang-auto"></code></pre>
</li>
<li>CategoryInfo : ObjectNotFound: (copycon:String) [], CommandNot<br>
FoundException</li>
<li>FullyQualifiedErrorId : CommandNotFoundException</li>
</ul>
<p>Is there an alternate command to use in Windows? Couldn’t find it on here. Thanks.</p>

**Comment by francisfuzz** [comment, marked as answer]
<p><a class="mention" href="/jwillens">@jwillens</a> Thanks for raising this question about Git here! I searched the <a href="https://help.github.com/en" rel="nofollow noopener">GitHub.com Help documentation</a> but couldn’t find such a guide containing the “Copycon” command. We suggest getting in touch with the maintainers or owners for that beginning course for Git on Windows and asking them about this as this doesn’t sound like something we (GitHub) authored nor maintains.</p>
<p>Alternatively, it may be worth opening an issue in the Git for Windows’ repository hosted on <a href="http://GitHub.com" rel="nofollow noopener">GitHub.com</a> and asking about this if that’s where you got this information from:</p>
<p><a href="https://github.com/git-for-windows/git/issues/2287" rel="nofollow noopener">https://github.com/git-for-windows/git/issues</a></p>
<p>On a cursory search, it sounds like someone else has asked about this and received a reply for:</p>
<p><a href="https://github.com/git-for-windows/git/issues/2287" rel="nofollow noopener">https://github.com/git-for-windows/git/issues/2287</a></p>
<p>I hope this helps!</p>

**Comment by jwillens** [comment]
<p>Thanks very much. I found it on the Lynda “Learning GitHub” course by Aaron Stewart, which links to the exercises in the GitHub Learning Lab. I’ll follow up there.</p>

**Reply by ghost** [reply]
its on LinkedIn Learning, under the section "Learning Githut."  Its actually linked to a couple of exercises listed on lab.github.com, which are also referred to by the instructor in the videos for reference and for practice to get better using Git Bash Command Line.  

https://www.linkedin.com/learning/learning-github/prepare-your-project?autoplay=true
